Directory.Packages.props

```diff
@@ -39,9 +39,9 @@
         <PackageVersion Include="System.Security.Cryptography.Xml" Version="8.0.1" /> <!-- Transitive dependency -->
         <PackageVersion Include="System.Text.Encodings.Web" Version="8.0.0" /> <!-- Transitive dependency -->
         <PackageVersion Include="System.Text.RegularExpressions" Version="4.3.1" /> <!-- Transitive dependency -->
-        <PackageVersion Include="xunit" Version="2.8.1" />
-        <PackageVersion Include="xunit.abstractions" Version="2.0.3" />
-        <PackageVersion Include="xunit.assert" Version="2.8.1" />
-        <PackageVersion Include="xunit.runner.visualstudio" Version="2.8.1" />
+        <PackageVersion Include="xunit.v3" Version="3.0.1" />
+        <PackageVersion Include="xunit.v3.assert" Version="3.0.1" />
+        <PackageVersion Include="xunit.v3.extensibility.core" Version="3.0.1" />
```

Comment:
Is the xunit.v3.extensibility.core PackageVersion needed? I didn't see any matching PackageReference items in the projects.